### ISSUE TITLE
Timeless events are missing

### ISSUE THREAD

**Issue by Rjs37** [antoniandre/vue-cal-v4#308]
From what I can see on the demo (and when I tested the feature locally) the timeless events no longer seem to be working. Potentially related to the changes around date props? I don't see any errors happening though. 

I've attached a screenshot of what I'm currently seeing on the demo.

![Selection_057](https://user-images.githubusercontent.com/1931638/85931916-b5685780-b8bf-11ea-8838-11c3a63315b6.png)

**Comment by antoniandre** [comment]
Thanks for reporting @Rjs37, I'll fix that soon.

**Comment by antoniandre** [comment]
Hi @Rjs37, @saulens22, this is now fixed in version 3.6.1.
Let me know if you still see an issue.

**Comment by saulens22** [comment]
Just updated to 3.6.1, and this bug persists. I'm using all-day events rendered in month view via slots.

vue-cal params:

```vue
<vue-cal
    active-view="month"
    locale="lt"
    events-count-on-year-view
    :events="events"
    :time="showTime"
    :today-button="false"
    :disable-views="['years', 'week', 'day']"
    :events-on-month-view="'short'"
    :watch-real-time="true"
    :show-week-numbers="true"
    :show-all-day-events="true"
    :selected-date="selectedDate"
    :on-event-click="eventClick"
    :on-event-dblclick="eventClick"
    style="min-height:300px"
    @view-change="changedView($event)">
    <template v-slot:event="{ event, view }">
    <div v-if="view='month'">
        <div class="target">
        custom render
        </div>
    </div>
    </template>
    <template v-slot:cell-content="{ cell, view, events }">
    <span class="vuecal__cell-date">{{ cell.content }}</span>
    <span
        class="vuecal__cell-events-count"
        v-if="['year'].includes(view.id) && events.length"
    >{{ events.length }}</span>
    </template>
</vue-cal>
```

Example event:
`{
start: 2020-09-29,
end: 2020-09-29,
title: 'xxxyyy',
...
}`

Workaround - when I get events from AJAX, I add:
`start = start + ' 00:00';
end = end + ' 23:59'`

**Comment by antoniandre** [comment]
Hi @saulens22,

I tried your code snippet and it does render the timeless events without time when `:time="showTime"` is set to false.
Be aware that if your events have no time information, but only a date, they will not render when showTime is true.

I don't have your full code so from what I have tried it looks good. Please give more details.

Also I don't understand why you are using all day events with timeless events, are they are already all-day if no time information.

Here is my test based on your snippet:

```html
  <vue-cal
    active-view="month"
    :events="events"
    :time="false"
    :disable-views="['years', 'week', 'day']"
    :events-on-month-view="'short'"
    watch-real-time
    show-week-numbers
    show-all-day-events
    style="min-height:300px">
    <template v-slot:event="{ event, view }">
      <div v-if="view === 'month'">
        <div class="target"> custom render </div>
      </div>
    </template>
    <template v-slot:cell-content="{ cell, view, events }">
      <span class="vuecal__cell-date">{{ cell.content }}</span>
      <span class="vuecal__cell-events-count" v-if="['year'].includes(view.id) && events.length" >{{ events.length }}</span>
    </template>
</vue-cal>
```

```js
    events: [
      {
        start: '2020-07-02',
        end: '2020-07-02',
        title: 'Event'
      }
]
```

**Comment by saulens22** [comment]
Thanks for your response @antoniandre 

Yes, it was a problem on my end. I set `:time` to variable for testing, but apparently it's not reactive - I need to update `:events` variable for `:time` to react 😅 

> Also I don't understand why you are using all day events with timeless events, are they are already all-day if no time information.

`allDay: true` is a leftover from an older version

**Comment by Rjs37** [comment]
@antoniandre As far as I can see it looks like they're still not working when part of the all day bar e.g. https://antoniandre.github.io/vue-cal/#ex--all-day-events (and that's actually the use-case I'm needing, sorry!)

Do you want me to create a separate issue for that? I thought they were going to be one and the same tbh!

**Comment by antoniandre** [comment]
@saulens22, no problem mate, to be frank I prefer the bug to be on your end. hahaha

Hi @Rjs37, this is now fixed in version 3.6.2.
Let me know if you are still facing an issue!

**Comment by Rjs37** [comment]
@antoniandre Yep that's done the job, cheers for the very quick turnaround! It works perfectly when using true, although with "short" I did come across a few quirks. I didn't realise I needed to explicitly set it as `:show-all-day-events="'short'"` rather than `:show-all-day-events="short"` - my mistake there :+1: 

But more importantly as soon as I enabled that short setting, it seems to then completely ignore (for all events) the custom event rendering/slot that I've defined. Looks like I can work around that by not using short, and checking specifically for the allDay property on the event, but it's still an interesting quirk that caught me by surprise.issues

It has also made me spot a width misalignment on my day columns for my timed events (when there's no scrollbar), but that's almost certainly going to be an issue on my end.
